### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by retronbv**
Resolves Readme being outdated

### Changes

Updated README

### Reason for changes

Out of date readme

### Tests
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
 <img src="https://raw.githubusercontent.com/SA-Userscript/ScratchAddons/master/images/icon.svg" alt="Scratch Addons logo" align="right" width="128px"></img>
 
-# Welcome to Scratch Addons' repository!
+# Welcome to Scratch Addons Userscript' repository!
```
Comment (wgyt) [inline README.md:3]:
```suggestion
# Welcome to Scratch Addons Userscript's repository!
```
There needs to be an s after the ' if the word doesn't end in s

File: README.md [hunk 2]
```diff
@@ -20,7 +16,7 @@
 
 ## About
 
-Scratch Addons is a WebExtension (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
+Scratch Addons Userscript is a userscript that has all the features of Scratch Addons' (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
```
Comment (wgyt) [inline README.md:19]:
```suggestion
Scratch Addons Userscript is a userscript that has all the features of Scratch Addons' extension. Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable. Learn more about Scratch Addons [here](https://github.com/scratchaddons/scratchaddons#readme)
```

**Review by wgyt — CHANGES_REQUESTED** [review]
See suggestions and remove lines 20-79

File: README.md [hunk 3]
```diff
@@ -20,66 +16,7 @@
...
-  1. Open `chrome://extensions` to open the Extension Management page by typing it into your address bar.
-  2. Enable Developer Mode by clicking the toggle switch next to Developer mode.
-  3. Click the `Load unpacked` button on the top and select the extension directory which has the `manifest.json` file in it.
-
-- Mozilla Firefox
-  1. Open `about:debugging` to open the add-ons page by typing it into your address bar.
-  2. Click `This Firefox` on the right-hand menu.
-  3. Click `Load Temporary Add-on...` and select the `manifest.json` file.
-
-## Contribute
```
Comment (wgyt) [inline README.md:80]: add the contribute section back

Comment (retronbv) [reply README.md:80]: o

**Comment by retronbv** [comment]
:shipit:

**Review by wgyt — APPROVED** [review]

**Pull request merged**